courses/pyladies/plan.yml

```diff
@@ -1,67 +1,99 @@
 - name: Instalace
   materials:
-  - link: lessons/beginners/cmdline
-  - link: lessons/beginners/install
-  - link: lessons/beginners/install_editor
-  - link: lessons/beginners/install_git
+  - name: Úvod do příkazové řádky
```

Comment:
And I don't think it's a problem, generally, if some items need more info than others.